frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/backing-storage-step/backing-storage-step.tsx

```diff
@@ -203,6 +203,17 @@ export const BackingStorage: React.FC<BackingStorageProps> = ({
     }
   }, [deployment, dispatch, sc, type]);
 
+  React.useEffect(() => {
+    /*
+     * Stores cluster's infrastructure which is then used in "securityAndNetwork" step.
+     * Better to store it in a global state rather than polling again for every required wizard step.
+     * In future, could be useful for other steps as well.
+     */
```

Comment:
Each step is a standalone component.
If anyway you think the resource is required to be used in more than one place then declare it in the main wizard component. "create-storage-system" .

Reply:
ack... will shift the polling to `create-storage-system` and pass it down as a prop to each required component, instead of adding a new state variable.